src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/LineBrush.java

```diff
@@ -22,7 +22,7 @@
 
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
-        if (parameters[1].equalsIgnoreCase("info")) {
+        if (parameters[0].equalsIgnoreCase("info")) {
             SnipeMessenger messenger = snipe.createMessenger();
             messenger.sendMessage(ChatColor.GOLD + "Line Brush instructions: Right click first point with the arrow. Right click with powder to draw a line to set the second point.");
```

Comment:
Putting this here for the entire project: Gunpower > powder. Especially in days of powdersnow and concrete powder. Replace on sight.